README.md

```diff
@@ -49,11 +51,30 @@ The build process is managed with [Make](https://www.gnu.org/software/make/).
 
 ### Make Commands
 
+* `load`: Load signed app onto the Ledger device
 * `clean`: Clean the build and output directories
 * `delete`: Remove the application from the device
-* `load`: Load signed app onto the Ledger device
 * `build`: Build obj and bin api artefacts without loading
-* `sign`: Sign current app.hex with CA private key
-* `deploy`: Load the current app.hex onto the Ledger device
+* `format`: Format source code.
 
 See `Makefile` for list of included functions.
+
+## How to get a transaction body computed by Ledger
+
+Ordinarily, Ledger computes a rolling hash of the serialized transaction body, but the body itself is not available. It is possible to acquire it from the development build by going through the following steps:
+
+1. [Install debug MCU](https://ledger.readthedocs.io/en/latest/userspace/debugging.html#application-debug) on your Ledger Nano S device.
+
+2. Install the debug version of Cardano app (see above).
+
+3. Install `usbtool` and turn on [console printing](https://ledger.readthedocs.io/en/latest/userspace/debugging.html#console-printing).
+
+4. Send a single `signTx` call to Ledger (e.g. by running `yarn test-integration --grep "<some-signTx-test>"`).
+
+5. After the call is processed, the terminal running console printing now contains all log messages resulting from that `signTx` call. (See the `TRACE*` macros.) Extract the transaction body logs (dumped by the function computing the rolling tx hash; you can identify them by function names following the pattern `blake2b_256_append*tx_body`) and merge them into a single hexstring like this:
+
+       const x = <content of the log file>
```

Comment:
it would perhaps be beneficial to clarify this is javascript and markdown supports js snippets as well so the snippet can be updated like this: 

```javascript
console.log("aaa")
```

https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code